mux/src/localpane.rs

```diff
@@ -478,6 +478,11 @@ impl Pane for LocalPane {
     }
 
     fn focus_changed(&self, focused: bool) {
+        if focused {
+            let mux = Mux::get();
+            mux.increment_pane_focus_serial();
```

Comment:
why does this happen only here and not for any other Pane impl?
There should probably be some comments here in the code to explain it!